### PR TITLE
kie-maven-plugin: use takari-maven-plugin packaging

### PR DESCRIPTION
**Pull request by psiroky**
* better alternative to the standard maven-plugin packaging.
   Requires less configuration in pom + by default adds support
   for integration testing

@mariofusco please take a look.

### CODE REVIEW

File: kie-maven-plugin/pom.xml [hunk 1]
```diff
@@ -116,6 +72,12 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.eclipse.sisu</groupId>
+          <artifactId>org.eclipse.sisu.inject</artifactId>
+        </exclusion>
```
Comment (psiroky) [inline kie-maven-plugin/pom.xml]: I am not 100% sure about this exclusion. It is needed to make the `takari-maven-plugin` lifecycle work and the tests are running fine as well. @mariofusco do you know if the `org.eclipse.sisu.inject` could be needed in some case not covered by the tests?

**Comment by psiroky** [comment]
Sorry, I referenced this PR in non-related commit. This has not been merged (yet) into any branch.

**Comment by mswiderski** [comment]
shall this be merged or still awaits @mariofusco review?

**Comment by psiroky** [comment]
I would like someone else to take a look. @mariofusco would you be able to quickly review this?

**Comment by mariofusco** [comment]
+1, ok to merge

**Pull request closed without merging**